requirements/edx/github.txt

```diff
@@ -79,7 +79,7 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.12#egg=XBlock==0.4.12
 git+https://github.com/edx/edx-ora2.git@1.1.9#egg=ora2==1.1.9
 -e git+https://github.com/edx/edx-submissions.git@1.1.1#egg=edx-submissions==1.1.1
 git+https://github.com/edx/ease.git@release-2015-07-14#egg=ease==0.1.3
-git+https://github.com/edx/i18n-tools.git@v0.3.2#egg=i18n-tools==v0.3.2
+git+https://github.com/edx/i18n-tools.git@v0.3.4#egg=i18n-tools==v0.3.4
```

Comment:
I do see your comment "Had to upgrade django-babel-underscore and i18n-tools dependencies to prevent i18n strings in underscore templates from compiling to dict with callback arg" but I'm unclear as to its effect in the non-CircleCI realm. 
I'm trying to avoid changes coming in through a PR entitled "Fixes successfully build on circleci" that are unrelated to building on CircleCI because that will make it more confusing for other devs down the road.